operator/pkg/central/values/translation/translation.go

```diff
@@ -231,6 +233,24 @@ func getCentralDBComponentValues(c *platform.CentralDBSpec) *translation.ValuesB
 	return &cv
 }
 
+func getTelemetryValues(t *platform.Telemetry) *translation.ValuesBuilder {
+	cv := translation.NewValuesBuilder()
```

Comment:
Looking at the file, this seems more in line with the naming (`telemetry values -> tv`):
```suggestion
	tv := translation.NewValuesBuilder()
```